src/ucp/api/ucp.h

```diff
@@ -1564,12 +1559,13 @@ ucs_status_t ucp_worker_fence(ucp_worker_h worker);
 /**
  * @ingroup UCP_WORKER
  *
- * @brief Flush all outstanding communication on the @ref ucp_worker_h "worker"
+ * @brief Flush outstanding one-sided operations on the @ref ucp_worker_h
```

Comment:
i would say that it guarantees memory delivery of rma and amo. and not mention tag matching.